models/User.js

```diff
@@ -11,7 +11,7 @@ const User = (sequelize, DataTypes) => {
   /**
    * @type {Sequelize.Model}
    */
-  const UserSchema = sequelize.define('user', {
+  const UserSchema = sequelize.define('users', {
```

Comment:
#### Critical

- Model definition should be singular not plural.

Reply:
done